base/broadcast.jl

```diff
@@ -96,7 +96,7 @@ dumpbitcache(Bc::Vector{UInt64}, bind::Int, C::Vector{Bool}) =
             # reverse-broadcast the indices
             @nexprs $nargs i->(I_i = newindex(I, imap_i))
             # extract array values
-            @nexprs $nargs i->(@inbounds val_i = _broadcast_getvals(A_i, I_i))
```

Comment:
Doesn't much matter, though I wonder if at some point we could get rid of `unsafe_getindex`.